mlflow/ml-package-versions.yml

```diff
@@ -96,12 +94,6 @@ tensorflow:
 keras:
   package_info:
     pip_release: "keras"
-    install_dev: |
-      # In keras >= 2.4.0, `keras` simply redirects to `tensorflow.keras`:
-      # https://github.com/keras-team/keras#multi-backend-keras-and-tfkeras
-      # Unpin `tf-nightly` once the following issue is addressed:
-      # https://github.com/tensorflow/tensorflow/issues/46429
-      pip install 'tf-nightly==2.5.0.dev20210112' 'keras>=2.4.0'
```

Comment:
@harupy tf-nightly appears to be incompatible with `keras>=2.4.0`, as seen in https://github.com/mlflow/mlflow/pull/4371/checks?check_run_id=2616128928. Given that TF Keras is the recommended way to use Keras, I propose dropping dev version testing for Keras. We still get TF Keras dev version testing from `tensorflow`. Thoughts?

Reply:
Sounds good to me.